public/admin/config.yml

```diff
@@ -5,6 +5,7 @@ media_folder: public/images
 public_folder: /images
 publish_mode: editorial_workflow
 display_url: https://www.wargabantuwarga.com
+local_backend: true
```

Comment:
Please remove this in your PR, so we would keep using the GitHub backend when it's merged to the `main` branch.

Reply:
ahh, sorry, removed.